tests/test_particleContainer.py

```diff
@@ -58,6 +77,25 @@ def soa_particle_container(Npart, std_geometry, distmap, boxarr, std_real_box):
 
     iseed = 1
     pc.init_random(Npart, iseed, myt, False, std_real_box)
+
+    # add runtime components: 1 real 2 int
+    pc.add_real_comp(True)
+    pc.add_int_comp(True)
+    pc.add_int_comp(True)
+
+    #   can be removed after
+    #     https://github.com/AMReX-Codes/amrex/pull/3615
+    pc.resize_runtime_real_comp(1, True)
```

Comment:
@atmyers for some reason, the `np` (POD vector) size of the new runtime real component is still zero after this:

In debugging:
```
0 x 2
1 y 2
2 z 2
3 a 2
4 b 2
5 c 2
6 d 2
7 e 2
8 f 0
```

Reply:
Merging to AMReX:
- https://github.com/AMReX-Codes/amrex/pull/3615
- https://github.com/AMReX-Codes/amrex/pull/3861